openmdao/core/problem.py

```diff
@@ -1302,7 +1302,7 @@ def check_totals(self, of=None, wrt=None, out_stream=_DEFAULT_OUT_STREAM, compac
         return data['']
 
     def compute_totals(self, of=None, wrt=None, return_format='flat_dict', debug_print=False,
-                       driver_scaling=False):
+                       driver_scaling=False, global_names=False):
```

Comment:
Why not use abs_names here to be consistent with the notion of absolute names?

Reply:
I agree that this is inconsistent with abs_names.  I was just copying the naming used in Driver._compute_totals.  We should probably change it across the board, but that would mean deprecating global_names in Driver._compute_totals.